### DISCUSSION TITLE
How are things going?

### DISCUSSION THREAD

**Discussion by edwja** [yousty/event_store_client#58] [Q&A] [answered]
Greetings!

We are considering using EventStoreDB in our Rails ecosystem and are interesting in using this gem as our client.  What is the status of this gem?  If this project is active, we’d be willing to help out as contributors.

Any information is appreciated.

Regards,

Andy Edwards
Merchants Bonding Company

**Comment by intale** [comment, marked as answer]
Hey @edwja. This gem is not maintained any more. However, it should be just fine with latest ruby/grpc gem. We will accept any potential contribution to this gem, so simply create a PR.

Why this gem was abandoned? We decided to go with [pg_eventstore](https://github.com/yousty/pg_eventstore/) since it is more flexible and more performant(it has some [limitations](https://github.com/yousty/pg_eventstore/blob/main/docs/how_it_works.md#database-architecture) though). You could try it by yourself as well before making a final decision regarding the database.

**Reply by edwja** [reply]
Hi @intale.  What kind of benchmarks are you using to compare the performance of `event_store_client` and `pg_eventstore`?

**Reply by intale** [reply]
Hey Andy. I wrote some scripts to compare PostgreSQL performance vs ES DB performance(I deleted them already, they existed only on my local PC). Basically I was comparing read and write operations. APIs of `event_store_client` and `pg_eventsore` are very similar, so you can write some simple comparison scripts by yourself.
